examples/tutorials/homework/lesson5/ddpg_quadrotor/train.py

```diff
@@ -104,6 +99,8 @@ def evaluate(env, agent, render=False):
 
 # 创建飞行器环境
 env = make_env("Quadrotor", task="hovering_control")
+env = ActionMappingWrapper(env)
```

Comment:
Please add a comment in Chinese for this line. It's the tutorial for beginners.

Reply:
done